internal/plugins/helm/v1/scaffolds/templates/makefile.go

```diff
@@ -40,13 +44,8 @@ func (f *Makefile) SetTemplateDefaults() error {
 	}
 
 	f.TemplateBody = makefileTemplate
-
 	f.IfExistsAction = file.Error
 
-	if f.Image == "" {
-		f.Image = "controller:latest"
-	}
```

Comment:
I see that we're explicitly setting `f.Image` in the scaffold calls, but with this defaulter removed, it is now possible to use `templates.Makefile` and end up with an invalid Makefile.

We either need to return an error if `f.Image` is not set OR we need to keep this as the default.

My preference would be to stay aligned with Kubebuilder's Go plugin, which leaves this conditional in so that `f.Image` is defaulted to `controller:latest` if it isn't explicitly set by a caller.